src/containers/Events.jsx

```diff
@@ -1,27 +1,49 @@
 import React, {Component} from "react";
 import EventList from "../components/EventList";
+import EventCalendar from "../components/EventCalendar";
+import {Container, Row, Col, Button} from "react-bootstrap";
 import "./../App.scss";
+import {getEvents} from "../modules/gcal";
+import "./styles/Events.scss";
 
 class Events extends Component {
+  constructor() {
+    super();
+    this.state = {
+      events: [],
+    };
+  }
+
+  componentDidMount() {
+    getEvents(events => {
+      this.setState({events});
+    });
+  }
+
   render() {
     return (
-      <div className="container">
-        <div className="container-left">
-          {" "}
-          <EventList />{" "}
-        </div>
-        <div className="container-right">
-          <div className="responsiveCal">
-            <iframe
-              title="se-soc-event-calendar"
-              className="calendar"
-              src="https://calendar.google.com/calendar/embed?src=lb6ttb1ks37ql5r8isuasgcdo8%40group.calendar.google.com&ctz=America%2FToronto&showPrint=0"
-              frameBorder="0"
-              scrolling="no"
-            ></iframe>
-          </div>
-        </div>
-      </div>
+      <Container>
+        <Row>
+          <Col sm={5}>
+            {" "}
+            <EventList events={this.state.events} />{" "}
+          </Col>
+          <Col sm={7}>
+            <div style={{height: "600px"}}>
+              <EventCalendar events={this.state.events} />
+            </div>
+            <div className="add-cal">
+              <Button
+                variant="secondary"
+                className="add-cal-btn"
+                href="https://calendar.google.com/calendar/ical/9lt5o1quln7c4u5d4mhm223pa4%40group.calendar.google.com/public/basic.ics"
```

Comment:
This link will download the calendar .ics (idk what type of access level it is)

Reply:
unfortunately I don't think .ics files update with new events, they just contain info from the current events on the calendar. Can you use ```https://calendar.google.com/calendar?cid=bGI2dHRiMWtzMzdxbDVyOGlzdWFzZ2NkbzhAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ``` instead? This should open google calendar and give the option to add the calendar to your google calendars (read only access).

Reply:
I like your method better :) & changed it to the link you provided.

fyi, from my testing, it seems like the ics does update with new events.

Reply:
the ICS does update with the current events when you download a new one, but if you have it downloaded and then I add a new event, you need to re-download the ICS to get the new events. It is a one time pull, not a subscription. It just contains a copy of all existing events from the calendar, and they are yours to edit/delete etc but changes won't be reflected to anyone else's versions, and you won't automatically get any changes made to the original events

Reply:
Ah you're right - I was _already subscribed_ to the calendar I was editing. I thought I was editing the ics one when in reality it wasn't, my bad 😅